.doc_gen/metadata/cross_metadata.yaml

```diff
@@ -121,7 +121,11 @@ cross_LambdaScheduledEvents:
   synopsis: create an &LAMlong; function invoked by an &EVlong; scheduled event that
     sends anniversary text messages.
   languages:
-    JavaScript:
+     Java:
+      versions:
+        - sdk_version: 2
+          block_content: cross_LambdaAPIGateway_Java_block.xml
```

Comment:
I think you meant to add this to the cross_LambdaAPIGateway example instead of the scheduled events one.

Reply:
fixed